asreview/datasets.py

```diff
@@ -41,9 +55,154 @@ def to_dict(self):
     def __str__(self):
         return pretty_format(self.to_dict())
 
+    def find(self, data_name):
+        if data_name in self.aliases:
+            return self
+        return None
+
+    def list(self, latest_only=True):
+        return [self]
+
+
+class BaseVersionedDataSet():
+    def __init__(self, base_dataset_id, datasets=None, url=None):
+        if datasets is None and url is not None:
+            index_file = url+"/index.json"
+            with open(index_file, "r") as f:
```

Comment:
Not possible to open a url with `open`. This is required. The plugin should be a URL, not a relative filepath.